modules/tinymce/src/core/test/ts/browser/dom/ControlSelectionTest.ts

```diff
@@ -224,8 +224,6 @@ describe('browser.tinymce.core.dom.ControlSelectionTest', () => {
   it('TINY-9731: data-mce-selected should appear on selected details element', () => {
     const editor = hook.editor();
     editor.setContent(`<details><summary>hoy</summary><p>tiny</p></details>`);
-    TinyAssertions.assertContentPresence(editor, { 'details[data-mce-selected="1"]': 0 });
-    TinySelections.setCursor(editor, [ 0, 0 ], 0);
```

Comment:
The details is automatically selected at setContent due to nodeChange event, so this is not needed.